.travis.yml

```diff
@@ -6,17 +6,20 @@ rvm:
   - 2.3.8
   - 2.4.5
   - 2.5.3
+  - 2.6.4
 
 gemfile:
   - gemfiles/rails_4.1.gemfile
   - gemfiles/rails_4.2.gemfile
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 
 before_install:
 - gem update --system --no-doc
-- gem install bundler
+- gem install -v 1.17.3 bundler --no-rdoc --no-ri
```

Comment:
The build is failing but I'm not sure if its getting to this point. Could you please update the `before_install` to the following and push to see if it helps at all?

```yaml
before_install:
  - gem update --system 
  - gem install bundler:1.17.3
```

Reply:
Yikes, sorry about that @cparker-ajla. Could you try this instead:
```yaml
  - gem install bundler -v "1.17.3"
```

Reply:
Thank you for all your help getting past this error @javierjulio!